flask_app/paths.py

```diff
@@ -1,10 +1,14 @@
 import os
 
 COMBADGE_PATH_V1 = os.path.join(os.path.dirname(__file__), "..", "webapp", "dist", "assets", "combadge.py")
-COMBADGE_PATH_V2 = os.path.join(os.path.dirname(__file__), "..", "combadge", "target", "x86_64-unknown-linux-musl", "release", "combadge")
+COMBADGE_ASSETS_DIR = os.path.join(os.path.dirname(__file__), "..", "combadge_assets")
+COMBADGE_PATH_LINUX = os.path.join(COMBADGE_ASSETS_DIR "combadge_linux")
+COMBADGE_PATH_WINDOWS = os.path.join(COMBADGE_ASSETS_DIR "combadge_windows")
+COMBADGE_PATH_DARWIN = os.path.join(COMBADGE_ASSETS_DIR "combadge_darwin")
```

Comment:
Isn't there a comma missing after `COMBADGE_ASSETS_DIR`?

Also, can't we just append the OS to the path in lower case?

Reply:
Fixed